VocaDbWeb/wwwroot/Content/Styles/Overrides.less

```diff
@@ -222,18 +222,48 @@ ul.nav.nav-pills {
 	margin-bottom: 0;
 }
 
+legend {
+	margin-bottom: 0;
+	border: none;
+	line-height: 1em;
+}
+
+fieldset {
+	padding: 0.4em 0.4em;
+	margin: 1.4em 0.4em;
+	border: none;
+	border-bottom: 1px solid #7da5b982;
+}
+
+.legend-note {
+	font-weight: 500;
+	font-size: 13px;
+	float: right;
+	padding: 4px 7px;
+}
+
+.alert {
+	margin-bottom: 0;
+	padding: 8px 14px;
+}

```

Comment:
It seems that this causes unwanted side effects. For example, compare https://vocadb.net/Album/Create and https://beta.vocadb.net/Album/Create. I'm not sure if there would be any unwanted side effects other than this. It's surprisingly difficult to style HTML with CSS. I think it's better not to merge #820 and #839 into main for now.